code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -28,6 +28,9 @@
 	GLOB.xeno_mob_list += src
 	round_statistics.total_xenos_created++
 
+	if(is_centcom_level(z) && hivenumber == XENO_HIVE_NORMAL)//so admins can safely spawn xenos in Thunderdome for tests.
```

Comment:
outside the scope, also shitcode and will cause issues with ERTs

Reply:
This is how the queen was handled upon spawn.
What do you suggest?

Reply:
because queens werent spawned by erts, normal xenos are. just don't do this.